src/grafana_dashboards/gocert.json

```diff
@@ -0,0 +1,488 @@
+        "title": "Server Performance",
+        "type": "stat"
+      },
+      {
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PF33313B8A1E04527"
+        },
+        "fieldConfig": {
+          "defaults": {
+            "color": {
+              "mode": "palette-classic"
+            },
+            "custom": {
+              "axisCenteredZero": false,
+              "axisColorMode": "text",
+              "axisLabel": "",
+              "axisPlacement": "auto",
+              "barAlignment": 0,
+              "drawStyle": "line",
+              "fillOpacity": 0,
+              "gradientMode": "none",
+              "hideFrom": {
+                "legend": false,
+                "tooltip": false,
+                "viz": false
+              },
+              "lineInterpolation": "linear",
+              "lineWidth": 1,
+              "pointSize": 5,
+              "scaleDistribution": {
+                "type": "linear"
+              },
+              "showPoints": "auto",
+              "spanNulls": false,
+              "stacking": {
+                "group": "A",
+                "mode": "none"
+              },
+              "thresholdsStyle": {
+                "mode": "off"
+              }
+            },
+            "mappings": [],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "green",
+                  "value": null
+                },
+                {
+                  "color": "red",
+                  "value": 80
+                }
+              ]
+            }
+          },
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 7,
+          "w": 12,
+          "x": 7,
+          "y": 9
+        },
+        "id": 8,
+        "options": {
+          "legend": {
+            "calcs": [],
+            "displayMode": "list",
+            "placement": "bottom",
+            "showLegend": true
+          },
+          "tooltip": {
+            "mode": "single",
+            "sort": "none"
+          }
+        },
+        "pluginVersion": "9.5.3",
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "PF33313B8A1E04527"
+            },
+            "editorMode": "builder",
+            "expr": "sum_over_time(http_requests_total{juju_charm=\"gocert-k8s\"}[1m])",
+            "legendFormat": "{{method}} - {{code}}",
+            "range": true,
+            "refId": "A"
+          }
+        ],
+        "title": "Network Performance",
+        "type": "timeseries"
+      },
+      {
+        "collapsed": false,
+        "gridPos": {
+          "h": 1,
+          "w": 24,
+          "x": 0,
+          "y": 16
+        },
+        "id": 4,
+        "panels": [],
+        "title": "Logs",
+        "type": "row"
+      },
+      {
+        "datasource": {
+          "type": "loki",
+          "uid": "P2C167435204F2BB6"
```

Comment:
I'm pretty sure those uid's shouldn't be hardcoded, you can use variables like `"uid": "${prometheusds}"` for prometheus. 

Reference:
- https://github.com/canonical/vault-k8s-operator/blob/main/src/grafana_dashboards/vault.json

Reply:
I just chose the data source from the dropdown list that the dashboard shows automatically. I will try to find out how to do this.

Reply:
This is a unique id that will change for each deployment, for instance when I try to deploy the charm from your branch, I get this error because my prometheus datasource UID is different:
![image](https://github.com/user-attachments/assets/44a44488-f8ab-4b18-9866-df8ced667792)

Reply:
fixed